mne/io/tests/test_raw.py

```diff
@@ -1,14 +1,71 @@
 # Generic tests that all raw classes should run
 from os import path as op
-from numpy.testing import assert_allclose
+from numpy.testing import (assert_allclose, assert_array_almost_equal,
+                           assert_array_equal)
+from nose.tools import assert_equal
 
 from mne.datasets import testing
 from mne.io import Raw
+from mne.utils import _TempDir
+
+
+def _test_raw_object(reader, test_preloading, **kwargs):
+    """Test reading, writing and slicing of raw classes.
+
+    Parameters
+    ----------
+    reader : function
+        Function to test.
+    test_preloading : bool
+        Whether not preloading is implemented for the reader. If True, both
+        cases and memory mapping to file are tested.
+    **kwargs :
+        Arguments for the reader.
+
+    Returns
+    -------
+    raw : Instance of Raw
+        A preloaded Raw object.
+    """
+    tempdir = _TempDir()
+    raws = list()
+    raws.append(reader(**kwargs))
+    picks = [1, 3, 5]
+    bnd = 100
```

Comment:
Look here for code you can copy-paste:

https://github.com/mne-tools/mne-python/pull/2628/files#diff-89350124c3b525af7b8aabdf157ae7ecR145

The two lines that are reader-specific here are the `assert_equal` lines involving `bnd`, the others could (and probably should) be used for any reader.